Assets/PatchKit Patcher/Scripts/AppData/Local/ZipUnarchiver.cs

```diff
@@ -69,8 +70,13 @@ public void Unarchive(CancellationToken cancellationToken)
         private void UnarchiveEntry(ZipEntry zipEntry)
         {
             DebugLogger.Log(string.Format("Unarchiving entry {0}", zipEntry.FileName));
-
-            zipEntry.Extract(_destinationDirPath, ExtractExistingFileAction.OverwriteSilently);
+            MemoryStream memoryStream = new MemoryStream();
+            string destPath = Path.Combine(_destinationDirPath, HashCalculator.ComputeMD5Hash(zipEntry.FileName));
```

Comment:
Why it doesn't use `MapHashExtractedFiles` ?

Reply:
sorry, mistake